java2typescript-jackson/src/test/resources/java2typescript/jackson/module/WriterPreferencesTest.enumToStringLiteralType.d.ts

```diff
@@ -3,5 +3,11 @@ export type Enum =
     | "VAL2"
     | "name";
 
+export interface TestClassHasMapWithEnumKey {
+    enumKeyMap: { [key in Enum ]: string;};
+    stringKeyMap: { [key: string ]: string;};
+    intKeyMap: { [key: string ]: string;};
```

Comment:
I think it is better to not mix different issues, specially when
1) You only fix one of them
2) the expected test output contains invalid result and test still passes